fe/tailwind.config.js

```diff
@@ -39,12 +39,28 @@ module.exports = {
                     "linear-gradient(180deg, #F0F0F0 0%, #F5DED4 100%)"
             },
             spacing: {
+                "8.5": '2.125rem',
                 "8xl": "96rem",
-                "9xl": "128rem"
+                "9xl": "128rem",
+                "23.25" : "5.813rem",
+                "44.5" : "11.156rem",
             },
             borderRadius: {
                 "4xl": "2rem"
-            }
+            },
+            height: {
+                "30.5": '7.625rem',
+                "42.5": '10.563rem',
+                "100.5": '25.125rem',
+            },
+            width: {
+                "19": '4.75rem',
+                "71.75": '17.938rem',
+                "85": '21.25rem',
+            },
```

Comment:
Update these as discussed.
Please add a comment on Figma so I can update it there as well.